packages/blade/src/components/Typography/Heading/Heading.tsx

```diff
@@ -111,13 +115,16 @@ const getProps = <T extends { variant: HeadingVariant }>({
     }
     props.fontSize = 75;
     props.lineHeight = 50;
-    props.as = isPlatformWeb ? 'h6' : undefined;
+    props.as = isPlatformWeb ? 'p' : undefined;
   }
```

Comment:
Ek ternary keliya 6 lines? doesn't seem very readable to me. 

But i think ye isPlatformWeb chaihain hi nehi anymore since we are not throwing as prop error on native.